lib/widgets/mxc_image.dart

```diff
@@ -56,23 +60,35 @@ class MxcImage extends StatefulWidget {
 class _MxcImageState extends State<MxcImage>
     with SingleTickerProviderStateMixin {
   static const String placeholderKey = 'placeholder';
-  static final Map<String, Uint8List> _imageDataCache = {};
+  static final MxcImageCacheMap _imageDataCache = {};
   Uint8List? _imageDataNoCache;
   bool isLoadDone = false;
 
   Uint8List? get _imageData {
     final cacheKey = widget.cacheKey;
-    final image =
-        cacheKey == null ? _imageDataNoCache : _imageDataCache[cacheKey];
+    final image = cacheKey == null
```

Comment:
it already cache in the database, does it really need to cache further more ?

Reply:
it only cache file in database if you pass uri instead event, avatar is the case pass uri